build/windows/build_windows.py

```diff
@@ -29,14 +29,16 @@
 parser = argparse.ArgumentParser(description='builder for Windows')
 parser.add_argument('-m', '--mode', help='build mode: Debug or Release')
 parser.add_argument('-a', '--arch', help='architecture: x86 or x64')
-parser.add_argument('-t', '--tool', help='platform toolset: v140, v141...')
+parser.add_argument('-t', '--tool', help='platform toolset: v140, v141, v142, v143...')
+parser.add_argument('-s', '--stdcpp', help='C++ standard: 11, 14, 17, 20...')
 parser.add_argument(      '--dll',  help='build DLL', action='store_true')
 
 args = parser.parse_args()
 
 BUILD_MODE = 'Debug'
-ARCH = 'x64'
-TOOLSET = args.tool
+ARCH = 'x86'
```

Comment:
I think by default should x64 as it's more popular

Reply:
https://github.com/heroiclabs/nakama-cpp/pull/52/commits/501c729d50332d81a72eb7e323c0c35412d96815